tests/Persisters/FilesystemTest.php

```diff
@@ -15,6 +15,13 @@
  */
 class FilesystemTest extends TestCase
 {
+    /**
+     * The path to the test file.
+     *
+     * @var string
+     */
+    const PATH = __DIR__ . '/test.persistable';
```

Comment:
will `__DIR__` always have write permissions on all systems?

Reply:
Interesting, I think I'll try that

Thanks!

Reply:
The cleanest solution I found for this last week was defining a constant in the global scope in constants.php so you could use 'TMPDIR' in place of '__DIR__'


https://github.com/RubixML/RubixML/pull/106#discussion_r481751711